### PR TITLE
Improve virtualenv description

### LINKED ISSUE #16: virtualenv description gently misleading
> This is essentially a 3rd party package that duplicates the standard libary venv module

is not exactly true these days; `virtualenv` 20.0.0+ has various tricks up its sleeve which make creating virtualenvs much faster than the stock module.

### PR DESCRIPTION
**Pull request by akx**
Closes #16

### CODE REVIEW

**Comment by akx** [comment]
Ping @cs01 :)

**Comment by jhermann** [comment]
BTW, for me the even larger advantage is that venv survives updates of the base Python (a frequent problem with python-minimal package updates before).

File: frontend/src/initialToolData.tsx [hunk 1]
```diff
@@ -146,7 +146,7 @@ const _initialToolData: Tool[] = [
     features: ['manual virtual environment creation'],
     name: 'virtualenv',
     toolDescription:
-      'A tool for creating isolated virtual Python environments. This is essentially a 3rd party package that duplicates the standard libary venv module. It predates venv, and works for all Python versions, including < 3.5 when venv was introduced.',
+      'A tool for creating isolated virtual Python environments. This is a predecessor of the Python 3.5+ standard library venv module, with improvements such as vastly faster virtualenv creation added in subsequent versions.',
```
Comment (cs01) [inline frontend/src/initialToolData.tsx:149]: I would like to be more clear that venv should be preferred unless using an old version of python or if speed is a concern.

Comment (cs01) [reply frontend/src/initialToolData.tsx:149]: And any other caveats such as the one @jhermann mentioned

Comment (akx) [reply frontend/src/initialToolData.tsx:149]: Why do you think venv "should be preferred"?

Comment (cs01) [reply frontend/src/initialToolData.tsx:149]: Because it is part of the standard library and it works fine.

Comment (jhermann) [reply frontend/src/initialToolData.tsx:149]: Because more general awesomeness.

Comment (ofek) [reply frontend/src/initialToolData.tsx:149]: Actually, I think `virtualenv` is much more preferable due to speed and active maintenance.

**Pull request merged**